{"uuid": "4a294207-57e2-4bad-b6f6-6197e4d81add", "children": ["21f33ab5-5466-4019-9d30-895aa29fd7d7", "5e1df570-dfc8-40d6-aecd-fcd2a9a6dbd6", "0bc62435-fa47-4aed-8e6d-ccbe88022c0e", "edf8dacb-4ce6-40e1-b8ae-ffbc6654ef0b", "e6a42df9-43cd-4742-8c98-f3fa2daef1e7", "2cb5c982-8e1e-4324-b9ab-0b7f6b14ddf6"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370797, "stop": 1699988370797}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370807, "stop": 1699988370807}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370814, "stop": 1699988370814}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370820, "stop": 1699988370820}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370826, "stop": 1699988370826}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988370833, "stop": 1699988370833}], "start": 1699988370833, "stop": 1699988370837}